app/src/main/java/com/example/easycloset/HomeFragment.java

```diff
@@ -27,24 +31,43 @@
 public class HomeFragment extends Fragment {
 
     private Weather weather;
-    private ImageView search;
     private EditText etCity;
+    private ImageButton btnSearch;
     private TextView tvCity, tvCountry, tvTemp, tvForecast, tvHumidity, tvMinTemp, tvMaxTemp, tvSunrise, tvSunset;
+    private ProgressDialog progressDialog;
+    private double latitude;
+    private double longitude;
+    private LocationRequest mLocationRequest;
```

Comment:
you don't use mPrefix in other variable names, it is probably copy/paste, make sure you have same style everywhere

Reply:
fixed